docs/pages/enterprise/hsm.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: Teleport HSM Support
+description: How to configure Hardware Security Modules to manage your Teleport CA private keys
+h1: Teleport HSM Support
+---
+
+Teleport can be configured to use a connected HSM to create and hold CA private
+keys.
+
+While most PKCS#11 HSMs should be supported, we test with AWS CloudHSM,
+YubiHSM2, and SoftHSM2.
+
```

Comment:
Please follow the structure similar to the articles here:

https://github.com/gravitational/teleport/blob/master/docs/pages/setup/guides/fluentd.mdx

## Prerequisites

- Teleport v(=teleport.version=) Cloud, Open Source or Enterprise
- Teleport admin tool `tctl` configured to access the cluster.
- Fluentd version v(=fluentd.version=).
- Docker version v(=docker.version=).

```code
$ terraform version
# Terraform v(=terraform.version=)
```

```code
$ tctl version
# Teleport v(=teleport.version=) go(teleport.golang)
#
# Test connectivity and admin permissions
$ tctl get roles
```

Create a folder `fluentd` to hold configuration and plugin state:

```code
$ mkdir -p event-handler
$ cd event-handler
```

## Step 1/n. Do this